test/sharness/t0139-swarm-rcmgr.sh

```diff
@@ -40,9 +40,35 @@ test_expect_success 'disconnected: swarm stats requires running daemon' '
   test_should_contain "missing ResourceMgr" actual
 '
 
+# test sanity scaling
+test_expect_success 'set very high connmgr highwater' '
+  ipfs config --json Swarm.ConnMgr.HighWater 1000
+'
+
+test_launch_ipfs_daemon
+
+test_expect_success 'conns and streams are above 2000' '
+  ipfs swarm limit system --enc=json | tee json &&
+  [ "$(jq -r .ConnsInbound < json)" -ge 2000 ] &&
+  [ "$(jq -r .StreamsInbound < json)" -ge 2000 ]
+'
+
+test_kill_ipfs_daemon
+
+test_expect_success 'set previous connmgr highwater' '
+  ipfs config --json Swarm.ConnMgr.HighWater 96
+'
+
+test_launch_ipfs_daemon
+
+test_expect_success 'conns and streams are above 800' '
+  ipfs swarm limit system --enc=json | tee json &&
+  [ "$(jq -r .ConnsInbound < json)" -ge 800 ] &&
+  [ "$(jq -r .StreamsInbound < json)" -ge 800 ]
+'
+
```

Comment:
We should avoid adding more tests on sharness, even more after all the efforts we are putting into migrating all of this into standard Go tests: https://github.com/ipfs/kubo/tree/master/test/cli

Reply:
I don't want to get more work (rewriting rcmgr tests to the new test framework).

Reply:
I propose we compromise :)

If you don't have time to rewrite/move tests (like here, don't want to block release on this), 
then create new issue for that work and add it to the next iteration (Kubo 0.19). 


I did that for cid.contact sharness in https://github.com/ipfs/kubo/issues/9562

Reply:
https://github.com/ipfs/kubo/issues/9573